src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -497,17 +497,9 @@ internal bool RuntimeSupportsByRefLikeGenerics
         }
 
 #nullable enable
-        // PROTOTYPE: Remove when we have a runtime with support
-        private bool _overrideRuntimeSupportsAsyncMethods;
-        internal void SetOverrideRuntimeSupportsAsyncMethods()
-        {
-            _overrideRuntimeSupportsAsyncMethods = true;
-        }
-
         // Keep in sync with VB's AssemblySymbol.RuntimeSupportsAsyncMethods
         internal bool RuntimeSupportsAsyncMethods
-            => RuntimeSupportsFeature(SpecialMember.System_Runtime_CompilerServices_RuntimeFeature__Async)
-               || _overrideRuntimeSupportsAsyncMethods;
+            => GetSpecialType(InternalSpecialType.System_Runtime_CompilerServices_AsyncHelpers) is { TypeKind: TypeKind.Class, IsStatic: true };
```

Comment:
Should we remove the runtime feature entry from the design doc?

Reply:
What was the resolution?

Reply:
It was removed.